packages/tpetra/scripts/PerformanceTesting/crusher/configureBuild.sh

```diff
@@ -58,14 +88,17 @@ ARGS=(
     -D MPI_EXEC:STRING="srun"
     #-D CMAKE_CXX_FLAGS:STRING="-I${ROCM_PATH}/include"
     -D Trilinos_EXTRA_LINK_FLAGS:STRING="-L${ROCM_PATH}/lib -lamdhip64"
-    -D CMAKE_CXX_COMPILER:PATH="$HIP_PATH/bin/hipcc"
+    #-D CMAKE_CXX_COMPILER:PATH="$HIP_PATH/bin/hipcc"
+    -D CMAKE_CXX_COMPILER:PATH="$HIP_CLANG_PATH/../../bin/hipcc"
     -D CMAKE_CXX_FLAGS:STRING="-I${CRAY_MPICH_DIR}/include"
-    -D CMAKE_C_COMPILER:PATH="$HIP_PATH/bin/hipcc"
+    #-D CMAKE_C_COMPILER:PATH="$HIP_PATH/bin/hipcc"
+    -D CMAKE_C_COMPILER:PATH="$HIP_CLANG_PATH/../../bin/hipcc"
     -D CMAKE_C_FLAGS:STRING="-I${CRAY_MPICH_DIR}/include"
     -D Trilinos_EXTRA_LINK_FLAGS:STRING="-L${CRAY_MPICH_DIR}/lib -lmpi -L${CRAY_MPICH_ROOTDIR}/gtl/lib -lmpi_gtl_hsa -L/${CRAY_BINUTILS_VERSION}/cce/x86_64/lib -lcraymath -lu -lf"
```

Comment:
`Trilinos_EXTRA_LINK_FLAGS` is used twice (first use is at line 90). Won't it override the first one?